pkg/profiles/profiles_test.go

```diff
@@ -80,7 +80,7 @@ spec:
       pathRegex: /route-1`,
 		},
 		{
-			err: errors.New("failed to validate ServiceProfile: error unmarshaling JSON: while decoding JSON: json: unknown field \"foo\""),
+			err: errors.New("failed to validate ServiceProfile: error unmarshalling JSON: while decoding JSON: json: unknown field \"foo\""),
```

Comment:
I think this error message actually comes from a dependency so we need to leave the "unmarshaling" spelling in this expectation.

Reply:
Yep. It took me a bit and I was starting to rework things to tolerate that, and then got interrupted.

It's unfortunate that the world is split on the spelling, but it won't heal anytime soon.

I'll have that tolerated w/ my next force-push.